### ISSUE TITLE
Invoking getUserMedia() before joinRoom() results in TypeError

### ISSUE THREAD

**Issue by letchoo** [Temasys/SkylinkJS#217]
Doing the following:

```
skylink.getUserMedia(function (error, success) {
   if (error) return;
  
   skylink.joinRoom();
});
```

Results in an error:
> TypeError: self._user is null

**Comment by letchoo** [comment]
Resolving this in `0.6.12` release.

**Comment by letchoo** [comment]
This has been resolved with [`0.6.12` release](https://github.com/Temasys/SkylinkJS/releases/tag/0.6.12).
